plugins/scaffolder-backend/src/lib/templating/filters.ts

```diff
@@ -27,7 +27,15 @@ export const createDefaultFilters = ({
 }): Record<string, TemplateFilter> => {
   return {
     parseRepoUrl: url => parseRepoUrl(url as string, integrations),
-    parseEntityRef: ref => parseEntityRef(ref as string),
+    parseEntityRef: (
+      ref,
+      defaultKind?: JsonValue,
```

Comment:
Curious if you think there's a better way to provide these optional args. I was looking at [nunjucks filters](https://github.com/mozilla/nunjucks/blob/master/nunjucks/src/filters.js#L395), and couldn't find a different example.

Reply:
Is it possible to mirror the api for `parseEntityRef` itself here?

`{{ parameters.entityRef | parseEntityRef({ defaultKind: 'asd' })` ?

Reply:
The following works:
```TypeScript
parseEntityRef: (ref, context?) => parseEntityRef(ref as string, context as JsonObject)
```


`{{ parameters.entityRef | parseEntityRef({ defaultKind:'asd' })`

Reply:
Yeah I think this is something similar to what we're looking for here.